community/values/src/main/java/org/neo4j/values/utils/PrettyPrinter.java

```diff
@@ -152,16 +152,22 @@ public void writePath( NodeValue[] nodes, EdgeValue[] edges )
     }
 
     @Override
-    public void beginPoint( CoordinateReferenceSystem coordinateReferenceSystem )
+    public void writePoint( CoordinateReferenceSystem crs, double[] coordinate ) throws RuntimeException
     {
-        stack.push( new PointWriter( coordinateReferenceSystem ) );
-    }
-
-    @Override
-    public void endPoint()
-    {
-        assert !stack.isEmpty();
-        append( stack.pop().done() );
+        append( "{geometry: {type: \"Point\", coordinates: [" );
+        for ( int i = 0; i < coordinate.length; i++ )
```

Comment:
Arrays.toString( coordinate ) and you will have array [a, b, c] without need to put commas yourself and being careful about that